#goal definition
sensor_msgs/Image input_img
float64 map_resolution
float64 Map_Origin_x
float64 Map_Origin_y
int32[] room_center_x
int32[] room_center_y
---
#result definition
int32[] room_number
int32 CenterPositionX
int32 CenterPositionY
---
#feedback definition